packages/cascara/src/ui/Form/context/FormProvider.js

```diff
@@ -7,11 +10,14 @@ import { ModuleContext, ModuleProvider } from '../../../modules/context';
 
 const propTypes = {
   children: pt.oneOfType([pt.node, pt.arrayOf(pt.node)]),
+  // A form needs to have a name to be accessible
+  // see: https://github.com/testing-library/dom-testing-library/issues/474
+  name: pt.string.isRequired,
```

Comment:
👍🏽 

Can we please also make a task to address this for Table? The Table also has a form so we need to be consistent with this support.

Reply:
sure thing, working on it.

Reply:
[FDS-297](https://espressive.atlassian.net/browse/FDS-297) created for this task 👍